app/assets/javascripts/application.js

```diff
@@ -63,7 +63,62 @@ $('.submit-new-exercise').on('click', function(event) {
       noticeWrapper.append('<div class="flash flash-error"><p class="notice">' +
         response.responseText + '</p></div>'
       );
-      $(window).scrollTop(0);
+      $('body').animate({
+        scrollTop: 0
+      }, 1000);
+    },
+  });
+});
+
+$('.exercises-table-list div.delete-icon').on('click', function(event) {
+  event.preventDefault();
+debugger;
+  var url = $('form#new_exercise')[0].action;
+  var workoutTable = $('.exercises-table-edit-page');
+//   var lastRow = $('.exercises-table-list');
+  var noticeWrapper = $('#notice_wrapper');
+  var flashMessage = $('.notice');
+  $.ajax({
+    type: 'DELETE',
+    url: url,
+    dataType: 'json',
+    data: JSON.stringify({id: $('#id').val()),
+    success: function(response) {
```

Comment:
Expected ')' and instead saw ':'.<br>Expected '}' to match '{' from line 81 and instead saw 'function'.<br>Bad invocation.<br>Expected ')' and instead saw '{'.<br>Missing semicolon.<br>'success' is not defined.<br>'response' is not defined.